Lib/enum.py

```diff
@@ -1826,7 +1837,7 @@ def convert_class(cls):
                     contained = value2member_map.get(member._value_)
                 except TypeError:
                     contained = None
-                    if member._value_ in unhashable_values:
+                    if member._value_ in unhashable_values or member.value in hashable_values:
```

Comment:
It looks long, maybe you can use **()** for line continuation.

```suggestion
                    if (member._value_ in unhashable_values or
                        member.value in hashable_values):
```
For other changes.

Reply:
See above comment about standardizing.